### DISCUSSION TITLE
Error after following tutorial on Prisma with Sveltekit.

### DISCUSSION THREAD

**Discussion by sadsunray** [prisma/prisma#19574] [Q&A]
Hello community,
I followed the tutorial on sveltekit with prisma on prisma.io but am stuck now on this error message.

No 'User' record(s) (needed to inline the relation on 'Post' record(s)) was found for a nested connect on one-to-many relation 'PostToUser'.

When I'm creating a draft with a user, that is not in the database, it throws me this error. (else everything is working just fine)
Can someone point me in the right direction where to look? I copied all the code samples from the website.
Thanks alot

**Comment by ludralph** [comment]
Hi @sadsunray 👋 

It looks like you are using the `connect` option to relate the new `Post` to a `User`. This depends on the specified `User` already existing and the error message sounds like it can't find it.

You could use [`connectOrCreate`](https://www.prisma.io/docs/reference/api-reference/prisma-client-reference#connectorcreate) which will create a new `User` if it can't find one.

**Reply by sadsunray** [reply]
Hi @ludralph 
Thanks for your answer. I will try that.
But what do I have to do when I just want to send back an error message which states, "user not in database".
I tried looking for that user in the page.Server.ts file in the /create folder, but this error gets no trigger.
Thanks for the help.

**Reply by ludralph** [reply]
Hi @sadsunray 

If you want to handle the scenario where the user does not exist in the database and return an error message, you can modify your code to explicitly check for the existence of the user before creating the post. If the user is not found, you can throw an error or return an appropriate response with the error message.

**Reply by ludralph** [reply]
Hey @sadsunray,

Do you have any more questions about this issue? If no, it would be really helpful if you could quickly **mark this Discussion as _answered_** to indicate that your issue has been resolved. 

In case you do have any outstanding questions, feel free to let us know and we'd be happy to help!
